src/apicompat/netstandard/netstandard.depproj

```diff
@@ -4,10 +4,12 @@
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <TargetFramework Condition="'$(TargetFramework)'==''">netstandard2.0</TargetFramework>
     <OutputPath>$(RestoredRefRootPath)</OutputPath>
-    <RestoreOutputPath>$(IntermediateOutputPath)$(TargetFramework)</RestoreOutputPath>
+    <RestoreOutputPath>$(IntermediateOutputPath)$(TargetFramework)\</RestoreOutputPath>
     <ProjectAssetsFile>$(RestoreOutputPath)/project.assets.json</ProjectAssetsFile>
+    <MSBuildProjectExtensionsPath>$(RestoreOutputPath)</MSBuildProjectExtensionsPath>
+    <_InitialMSBuildProjectExtensionsPath>$(MSBuildProjectExtensionsPath)</_InitialMSBuildProjectExtensionsPath>
```

Comment:
Did you still need this hack?  Makes me nervous we are using the internal property.

Reply:
I'm not sure how else to get around the issue, we do need to get the targets file imported for netstandard.library.

Reply:
I think the reason we did this hack was due to order of import / targetframework definition.  I believe you could find a different order of defining properties that would get this set eariler (if you move some things into Directory.Build.props).  Alternatively you could just ignore the targets and manually include the items.  I'd prefer either of those approaches over using the private property (I know we spent some time figuring this out but we shouldn't sink ourselves in the fragile solution because of that).

Reply:
I just pushed a commit that resolves the NetStandard.Library assets explicitly & removes this workaround